modules/cli-manager-installing.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-installing_{context}"]
+= Installing the {cli-manager}
+
+.Prerequisites
+
+* link:https://krew.sigs.k8s.io/docs/user-guide/setup/install[Krew is installed].
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+* You are using {product-title} version 4.17 or later.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Create the required namespace for the {cli-manager}.
+.. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
+.. Enter `openshift-cli-manager-operator` in the *Name* field and click *Create*.
++
+
+. Install the {cli-manager}.
+.. Navigate to *Operators* -> *OperatorHub*.
+.. Enter *{cli-manager}* into the filter box.
+.. Select the *{cli-manager}* and click *Install*.
+.. On the *Install Operator* page:
+... The *Update channel* is set to *tech preview*, which installs the latest technology preview release of the {cli-manager}.
```

Comment:
"Technology Preview" does always have to be capitalized, like the vale rule says

```suggestion
... The *Update channel* is set to *tech preview*, which installs the latest Technology Preview release of the {cli-manager}.
```